docs/paddlepaddle/how_to_create_paddlepaddle_model.md

```diff
@@ -0,0 +1,91 @@
+This document will show you how to train a model using the Paddle 2.0.
```

Comment:
Please format the document following the Markdown standard.

Reply:
done.